packages/did-resolver/src/memory-cache.ts

```diff
@@ -8,16 +8,18 @@ type CacheVal = {
 }
 
 export class MemoryCache extends DidCache {
-  public ttl: number
-  constructor(ttl?: number) {
+  public staleTTL
```

Comment:
I wonder if this is getting implicitly typed as `any`.
```suggestion
  public staleTTL: number
```

Reply:
ah man it was. good catch 👍